package.json

```diff
@@ -32,8 +32,9 @@
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
     "@alaskaairux/icons": "^4.27.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```

Comment:
Remove all references to the legacy version of design tokens. Also be sure to update `peerDependencies` too.